.github/workflows/release.yml

```diff
@@ -160,10 +83,22 @@ jobs:
         env:
           VERSION: ${{ github.ref }}
 
-      - name: Push to Docker Hub
-        uses: docker/build-push-action@v1
+      - name: Configure AWS credentials
+        uses: aws-actions/configure-aws-credentials@v1
         with:
-          repository: keratin/authn-server
-          username: ${{ secrets.DOCKERHUB_USERNAME }}
-          password: ${{ secrets.DOCKERHUB_TOKEN }}
-          tags: latest,${{ steps.version.outputs.number }}
+          aws-access-key-id: ${{ secrets.AWS_ACCESS_KEY_ID }}
+          aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+          aws-region: us-east-1
+
+      - name: Login to Amazon ECR
+        id: login-ecr
+        uses: aws-actions/amazon-ecr-login@v1
+
+      - name: Build, tag, and push image to Amazon ECR
+        env:
+          ECR_REGISTRY: ${{ steps.login-ecr.outputs.registry }}
+          ECR_REPOSITORY: keratin-sch
+          IMAGE_TAG: latest,${{ steps.version.outputs.number }}
```

Comment:
Is this something that we'll have to manually increment?

Reply:
We need to push our own tags